pkg/deploy/cmd/delete_test.go

```diff
@@ -120,41 +120,36 @@ func TestStop(t *testing.T) {
 				ktestclient.NewDeleteAction("deploymentconfigs", "default", "config"),
 			},
 			kexpected: []ktestclient.Action{
-				ktestclient.NewListAction("replicationcontrollers", "default", kapi.ListOptions{}),
+				ktestclient.NewListAction("replicationcontrollers", "default", kapi.ListOptions{LabelSelector: labels.SelectorFromSet(map[string]string{"openshift.io/deployment-config.name": "config"})}),
 				ktestclient.NewGetAction("replicationcontrollers", "", "config-1"),
 				ktestclient.NewListAction("replicationcontrollers", "", kapi.ListOptions{}),
 				ktestclient.NewGetAction("replicationcontrollers", "", "config-1"),
 				ktestclient.NewUpdateAction("replicationcontrollers", "", nil),
 				ktestclient.NewGetAction("replicationcontrollers", "", "config-1"),
-				ktestclient.NewGetAction("replicationcontrollers", "", "config-4"),
 				ktestclient.NewDeleteAction("replicationcontrollers", "", "config-1"),
 				ktestclient.NewGetAction("replicationcontrollers", "", "config-2"),
 				ktestclient.NewListAction("replicationcontrollers", "", kapi.ListOptions{}),
 				ktestclient.NewGetAction("replicationcontrollers", "", "config-2"),
 				ktestclient.NewUpdateAction("replicationcontrollers", "", nil),
 				ktestclient.NewGetAction("replicationcontrollers", "", "config-2"),
-				ktestclient.NewGetAction("replicationcontrollers", "", "config-5"),
```

Comment:
Don't get me started on this test.

On Sep 14, 2016, at 1:49 PM, David Eads <notifications@github.com> wrote:

In pkg/deploy/cmd/delete_test.go
<https://github.com/openshift/origin/pull/10777#discussion_r78799409>:

>  				ktestclient.NewDeleteAction("replicationcontrollers", "", "config-1"),
>  				ktestclient.NewGetAction("replicationcontrollers", "", "config-2"),
>  				ktestclient.NewListAction("replicationcontrollers", "", kapi.ListOptions{}),
>  				ktestclient.NewGetAction("replicationcontrollers", "", "config-2"),
>  				ktestclient.NewUpdateAction("replicationcontrollers", "", nil),
>  				ktestclient.NewGetAction("replicationcontrollers", "", "config-2"),
> -				ktestclient.NewGetAction("replicationcontrollers", "", "config-5"),

"carry the three, plus two more...", yep this was the line I was looking
for.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/origin/pull/10777/files/acd8aac3f65c5e5c1b3c50615fb713841440a693..2751f2c7bdfe639d4596ebf976b546c746918b79#r78799409>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/ABG_p6Ncis5CRu-l0ex-CxUSXgfZbaIHks5qqDOggaJpZM4JzJrj>
.